### ISSUE TITLE
[Bug] Show lost commits (Reflog) is broken in 1.66 update

### ISSUE THREAD

**Issue by carun** [ForkIssues/TrackerWin#1307]
It used to work previously in 1.65, but looks like it is broken in 1.66.

DAG from Fork with "Show lost commits" enabled:

![image](https://user-images.githubusercontent.com/471374/133139588-e550c07d-b29f-41f3-8c16-3ec5220e6ecc.png)

Equivalent git reflog via `git la`, where `la` and `lg` are aliases in `~/.gitconfig`

```
    lg = log --graph --decorate --all --date-order --date=format:'%Y-%m-%d %H:%M' --pretty=format:'%C(bold blue)%h%Creset %C(black bold)%ad%Creset %s %C(bold blue)%aN%Creset%C(auto)%d'
    la = !git lg $(git rev-list -g --all)
```

![image](https://user-images.githubusercontent.com/471374/133139688-c5a862c0-85da-4523-a1eb-350a8a2affaa.png)

**Comment by DanPristupov** [comment]
Reflog still works for me

BTW, equivalent is `git log --all --reflog`.

Ensure that filters are disabled or/and restart the application.

**Comment by carun** [comment]
Ah, interesting.. an unrelated branch was "hidden" from the view and that hid the reflog as well.

![image](https://user-images.githubusercontent.com/471374/133366477-16158ca5-2748-45b1-aa73-b00106586f19.png)

I can understand if a filter is applied and the reflog is hidden, because the intention is to filter and view only that specific branch.

But hiding a branch shouldn't hide reflog, isn't it?

**Comment by DanPristupov** [comment]
Fixed in Fork 1.94
